tests/target/configs/single_line_let_else_max_width/50.rs

```diff
@@ -0,0 +1,23 @@
+// rustfmt-single_line_let_else_max_width: 50
+
+fn main() {
+    let Some(a) = opt else {};
+
+    let Some(b) = opt else { return };
+
+    let Some(c) = opt else { return };
+
+    let Some(c) = opt else {
+        // a comment should always force the block to be multi-lined
+        return;
+    };
+
+    let Some(c) = opt else {
+        /* a comment should always force the block to be multi-lined */
+        return;
+    };
+
+    let Some(d) = some_very_very_very_very_long_name else {
+        return;
+    };
+}
```

Comment:
Could we include some additional cases at least here (with the val set to 50) that cover the other init expr scenarios?

E.g.

```rust
// Because the initializer expr is multi-lined the else is placed on it's own line.
    let Some(x) =
        some_really_really_really_really_really_really_really_really_really_long_name____H
    else {
        return;
    };
```

As well as the multilined init expr that starts on the same line as the assignment operator and ends with a sequence of one or more parens/brackets/braces?